index.html

```diff
@@ -9,42 +9,61 @@
   </head>
 
   <body>
-    <nav class="nav">
-      <a href="/" class="nav__link" data-link>List</a>
-      <a href="/" class="nav__link" data-link>Add New</a>
-      <a href="/" class="nav__link" data-link>Contact</a>
-    </nav>
-    <div class="books" id="books">
-      <h1>Awesome Books</h1>
-      <table>
-        <tr>
-          <th>Title</th>
-          <th>Author</th>
-          <th></th>
-        </tr>
-        <tbody id="lists"></tbody>
-      </table>
+    <div class="navigation">
+      <h1 class="logo"><a href="">Awesome Books</a></h1>
+      <nav class="nav">
+        <a href="#" class="nav__link list" >List</a>
+        <a href="#" class="nav__link addbooks" >Add New</a>
+        <a href="#" class="nav__link contacts" >Contact</a>
 
-       <h1>Add a New Book</h1>
-      <form id="form" class="form">
-        <input
-          type="text"
-          name="author"
-          id="atr"
-          placeholder="author"
-          class="atr"
-        />
-        <input
-          type="text"
-          name="title"
-          id="ttl"
-          placeholder="title"
-          class="ttl"
-        />
-        <button id="btn">Add</button>
-      </form>
+      </nav>
     </div>
+
+    <main class="main">
+      <div class="books" id="books">
+        <div class="wrapper">
+          <h1>Awesome Books</h1>
+          <table class="lists">
+            <tbody id="lists"></tbody>
+          </table>
+        </div>
+        <section class="addbook" id="addbook">
+          <h1>Add a Book</h1>
+          <form id="form" class="form">
+            <input
+              type="text"
+              name="author"
+              id="atr"
+              placeholder="author"
+              class="atr"
+            />
+            <input
+              type="text"
+              name="title"
+              id="ttl"
+              placeholder="title"
+              class="ttl"
+            />
+            <button id="btn">Add</button>
```

Comment:
The _add button_ does not have the appropriate **alignment**. Please note that we actually want our design to as much possible follow the [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) Kindly improve this aspect by ensuring that
- [ ] OPTIONAL
The _add button_ is **aligned to the right** with the form input, as can be seen in the wireframe design.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/183250220-e0ed1082-1928-4323-9ce2-a66d3b77609e.png)


**The wireframe design:**
![image](https://user-images.githubusercontent.com/12745474/143277255-658fcad7-519b-445d-b039-21a2187c6968.png)